ajax/libs/scion/package.json

```diff
@@ -1,17 +1,91 @@
 {
-  "name": "scion",
-  "filename": "scion-min.js",
-  "version": "0.0.3",
+  "name": "scxml",
```

Comment:
The name should be the same as folder name. If you wanna change this filed to `"name": "scxml",` the folder name should be `scxml`,